M0sense/RT-Thread/README.md

```diff
@@ -0,0 +1,114 @@
+---
+sys: rtthread
+sys_ver: null
+sys_var: standard
+
+status: basic
+last_update: 2024-12-05
+---
+
+# RT-Thread M0sense Test Report
+
+## Test Environment
+
+### Operating System Information
+- Source Code Link: https://github.com/RT-Thread/rt-thread
+- Reference Installation Document: https://github.com/RT-Thread/rt-thread/blob/master/bsp/bouffalo_lab/README.md
+- Toolchain: https://occ-oss-prod.oss-cn-hangzhou.aliyuncs.com/resource//1705395512373/Xuantie-900-gcc-elf-newlib-x86_64-V2.8.1-20240115.tar.gz
+
+### Hardware Information
+
+- Sipeed M0sense (BL702)
+- A USB A to C or C to C cable
+- One Dupont Line
+
+## Installation Steps
+
+The following steps are tested on Arch Linux, but should be applicable to all major Linux distributions.
+
+### Preparing the System Environment
+
+Install the required packages:
+
+```shell
+sudo apt install -y scons libncurses5-dev device-tree-compiler
+# or on Arch Linux: sudo pacman -S scons dtc ncurses 
+```
+
+Get the toolchain:
+
+```shell
+wget https://occ-oss-prod.oss-cn-hangzhou.aliyuncs.com/resource//1705395512373/Xuantie-900-gcc-elf-newlib-x86_64-V2.8.1-20240115.tar.gz
```

Comment:
Just FYI: I see this is the one from RTT's docs, have you tried the latest XuanTie toolchains (e.g. v3.0.1)?
https://www.xrvm.cn/community/download?id=4382928864901402624
(This won't block this PR though)

Reply:
> Just FYI: I see this is the one from RTT's docs, have you tried the latest XuanTie toolchains (e.g. v3.0.1)? https://www.xrvm.cn/community/download?id=4382928864901402624 (This won't block this PR though)

Not yet, but thanks for the info. Will update the rest of the RTT reports sometime later